gamemodes/terrortown/gamemode/server/sv_roleselection.lua

```diff
@@ -556,7 +563,7 @@ local function SelectForcedRoles(plys, selectableRoles)
 		-- if it's not a selectable role, continue
 		if not roleCount then continue end
 
-		local curCount = 0
+		local curCount = (selectedForcedRoles[subrole] or 0)
```

Comment:
Didn't notice this before, but this makes the order of forcing roles indepedent of each other.
I seem to miss so much on this topic. There are all these small little depndecies, which are easy to miss.

Hopefully that's the last little bug.

Reply:
And it's not. Just found another issue, when forcing baseroles first to the maximum and then force subroles, they come additionally, even if their baserole is maxed out...... oof